db/migrate/20190123210115_submission_changes_for0994.rb

```diff
@@ -0,0 +1,6 @@
+class SubmissionChangesFor0994 < ActiveRecord::Migration
+  def add
+    add_column(:education_benefits_submissions, :vettec, :boolean, default: false, null: false)
+    EducationBenefitsSubmission.update_all(vettec: false)
```

Comment:
I believe you can remove this `update_all` call, since the new column being added is set to `default: :false`.

Reply:
It's added to update value for all the existing records in the table. By adding new column, default would be false for any new records, not for existing ones, right?

Reply:
I believe that it does in fact set existing ones to the default, as well.  Perhaps you can try it out locally to confirm?

Reply:
Done

Reply:
@hpjaj : If the change looks fine, can you go ahead and approve if no other comments. thanks